### PR TITLE
Improve network load/save

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java [hunk 1]
```diff
@@ -150,23 +154,81 @@ public static ObjectHandle createEurostagTutorialExample1Network(IsolateThread t
...
+        return doCatch(exceptionHandlerPtr, () -> createCharPtrArray(new ArrayList<>(Importers.getFormats())));
+    }
+
+    @CEntryPoint(name = "getNetworkExportFormats")
+    public static ArrayPointer<CCharPointerPointer> getNetworkExportFormats(IsolateThread thread, ExceptionHandlerPointer exceptionHandlerPtr) {
+        return doCatch(exceptionHandlerPtr, () -> createCharPtrArray(new ArrayList<>(Exporters.getFormats())));
+    }
+
+    @CEntryPoint(name = "freeNetworkFormats")
+    public static void freeNetworkFormats(IsolateThread thread, ArrayPointer<CCharPointerPointer> formatsArrayPtr) {
```
Comment (sylvlecl) [inline java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java]: We should probably name it with a more generic "freeStringArray" or something like that

File: tests/test.py [hunk 2]
```diff
@@ -31,6 +31,14 @@ def test_run_lf(self):
         results = pp.loadflow.run_dc(n, parameters)
         self.assertEqual(1, len(results))
 
+    def test_get_import_format(self):
+        formats = pp.network.get_import_formats()
+        self.assertEqual(['CGMES', 'MATPOWER', 'IEEE-CDF', 'PSS/E', 'UCTE', 'XIIDM'], formats)
+
+    def test_get_export_format(self):
```
Comment (sylvlecl) [inline tests/test.py:46]: Can we add a test for parameters too ?

File: java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java (hunk 1 above)
Comment (geofjamg) [reply java/src/main/java/com/powsybl/python/PyPowsyblApiLib.java]: Done

File: tests/test.py (hunk 2 above)
Comment (geofjamg) [reply tests/test.py:46]: Added

**Review by sylvlecl — APPROVED** [review]

**Pull request merged**